src/commands/bundle-install.yml

```
@@ -27,6 +27,11 @@ steps:
   - restore_cache:
       keys:
         - '<< parameters.cache_key_prefix >>{{ checksum "/tmp/ruby-version" }}-{{ checksum "<< parameters.redmine_root >>/Gemfile" }}-{{ checksum "<< parameters.redmine_root >>/Gemfile.local" }}-{{ checksum "<< parameters.redmine_root >>/config/database.yml" }}'
+  - run:
+      name: key installation for chrome gpg key
+      command: |
+        wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo tee /etc/apt/trusted.gpg.d/google.asc >/dev/null
+        wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/google.gpg >/dev/null
```

[imo]one HTTP request is better than two HTTP requests, i think.

so, i recommend following changes:

```suggestion
        wget -q -O - https://dl.google.com/linux/linux_signing_key.pub |
          sudo tee /etc/apt/trusted.gpg.d/google.asc |
          gpg --dearmor |
          sudo tee /etc/apt/trusted.gpg.d/google.gpg >/dev/null
```

---

Thank you. I think your suggestion is better.
However I actually just followed chrome official way (see https://www.google.com/linuxrepositories/ )
Should we keep following official suggestion?

---

> just followed chrome official way

Ooh- official way! I see. Thank you for your response! :sparkles: